lib/file-view.coffee

```diff
@@ -14,6 +14,7 @@ class FileView extends HTMLElement
     @fileName.textContent = @file.name
     @fileName.dataset.name = @file.name
     @fileName.dataset.path = @file.path
+    @fileName.draggable = true
```

Comment:
It would be great if it were the .list-item parent element that were draggable, rather than the name - currently, you have to have the cursor over the text, otherwise the item can't be dragged, and if you're dragging it to a folder, it has to be dropped on the folder's text.